sql/donut.sql

```diff
@@ -173,32 +179,11 @@ CREATE VIEW group_house_membership AS (
         OR UPPER(pos_name) = UPPER('Social Member')
 );
 
--- Group Members Table
-CREATE TABLE group_members (
-    user_id  INT NOT NULL,
-    group_id INT NOT NULL,
-    send     BOOLEAN DEFAULT FALSE, -- Toggles whether or not this member can
-                                    -- send emails to group
-    control  BOOLEAN DEFAULT FALSE, -- Toggles whether or not this member has
-                                    -- admin control over group
-    receive  BOOLEAN DEFAULT TRUE,  -- Toggles if the member receives emails
-                                    -- From this group
-    PRIMARY KEY (user_id, group_id),
-    FOREIGN KEY (user_id) REFERENCES members(user_id),
-    FOREIGN KEY (group_id) REFERENCES groups(group_id)
-);
-
--- Group Positions Table
-CREATE TABLE group_positions (
-    pos_id      INT NOT NULL,
-    group_id    INT NOT NULL,
-    send        BOOLEAN DEFAULT FALSE, -- Toggles whether or not this position
-                                       -- can send emails to group
-    control     BOOLEAN DEFAULT FALSE, -- Toggles whether or not this position
-                                       -- has admin control over group
-    receive     BOOLEAN DEFAULT TRUE,  -- Toggles if this position receives
-                                       -- emails from this group
-    PRIMARY KEY (pos_id, group_id),
-    FOREIGN KEY (pos_id) REFERENCES positions(pos_id),
-    FOREIGN KEY (group_id) REFERENCES groups(group_id)
-);
+CREATE TABLE position_relations (
+    relation_id   INT  NOT NULL AUTO_INCREMENT,
+    pos_id_from   INT  NOT NULL,
+    pos_id_to     INT  NOT NULL,
```

Comment:
It looks like this table should be used in some helper functions but is not. Add a comment for what this table is.